changelog.d/10954.feature

```diff
@@ -0,0 +1 @@
+Add `--run-background-updates` option to `update_database` script.
```

Comment:
Feel free to add an attribution here!

```suggestion
Add `--run-background-updates` option to `update_database` script. Contributed by @Fizzadar.
```

Reply:
https://github.com/matrix-org/synapse/pull/10954/commits/57d6d44245e59e6bea421f2809caa3d69c761687